homeassistant/components/light/device_action.py

```diff
@@ -28,6 +28,9 @@
             toggle_entity.DEVICE_ACTION_TYPES
             + [TYPE_BRIGHTNESS_INCREASE, TYPE_BRIGHTNESS_DECREASE]
         ),
+        vol.Optional("brightness", default=100): vol.All(
```

Comment:
We shouldn't have a default. If it's not there, we shouldn't pass it to the service. That way a light will turn on in the previous brightness.

Reply:
So in the case where a user turned brightness down to 0% from say 10% (and we set the device off) wouldn't we want a minimum to be something like 10% because if they went to turn it on they would be surprised to see the device did not turn on because brightness was at 0% previously.  Sorry if this is offtopic but the default option made me think about this particular case.

Reply:
Also use a constant please.